Sam/Reworked.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AltarActivate : MonoBehaviour
+{
+
+    private GameObject objectOnAltar;
+    public GameObject Door { get; set; }
+    public PlayerItemSearch PlayerItemSearch { get; set; }
```

Comment:
Je hebt volgens de conventions een property in PascalCase gemaakt, maar je bent vervolgens in de rest van de code vergeten om de variable in PascalCase te schrijven waardoor je een error krijgt bij het uitvoeren van het script. Ook heeft je property dezelfde naam als je class PlayerItemSearch wat ook voor problemen kan zorgen.